kart/diff-view.html

```diff
@@ -103,8 +103,9 @@
             top: 2px;
         }
     </style>
-    <script id="kart-data">const DATA=${geojson_data};</script>
+    <script id="kart-data" type="application/json">${geojson_data}</script>
```

Comment:
what happens if geojson_data contains a string with `</script>` in it - does the nature of this json script tag ignore html in strings like that? or do we need to escape `/` characters or something?

django's `json_script` template filter might be worth checking out: https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#std-templatefilter-json_script

Reply:
```html
<!DOCTYPE html>
<html>
    <head>
        <script type="application/json">
            {
                "key": "</script><script>alert('xss')</script><script>{
            }
        </script>
    </head>
</html>
```

this alerts for me. so I think we need to escape `/` or something

Reply:
Good catch. Done, and added a test at last